sdk/src/auth/index.ts

```diff
@@ -133,12 +133,11 @@ export default class AWF_Auth implements AWF_IAuth {
       false,
     );
     return check.pipe(
-      tap(response => {
-        if (!response.data || response.data?.errors) {
-          const err = new Error('This ethereum key is not registered');
-          err.name = 'UserNotRegistered';
-          throw err;
-        }
+      tap(() => {
+        this._globalChannel.next({
```

Comment:
if this replaces the error throw case then it should still keep the guard rule

Reply:
I can add it, but from what ive seen it never threw here, getProfile has its own throw which always happened before this

Reply:
yes, but this is part of the `checkIfSignedUp` method here

Reply:
I still don't understand the use-case for the event emitted here

Reply:
this event will be emitted on every `checkIfSignedUp` method

Reply:
Hmmm good point. I used it to keep displaying the loader for connect address, until we know whether to stop the sign-up or not (already registered). But I guess this is something we can live without.

Reply:
you can still emit it but it needs to be aware of the response, for example you could also attach some data to the event like `isRegistered`

Reply:
Turns out there was no actual need for it in the end 🤦‍♂️🤦‍♂️ I'll let myself out